homeassistant/components/xiaomi_miio/remote.py

```diff
@@ -50,6 +50,11 @@
     {vol.Required(CONF_COMMAND): vol.All(cv.ensure_list, [cv.string])}
 )
 
+LED_ON_SCHEMA = vol.Schema({vol.Required(ATTR_ENTITY_ID): vol.All(str)})
```

Comment:
Define the service schema inline when registering the service by passing a dict. See the example in the docs previously linked.

Reply:
Inlined